codalab_service.py

```diff
@@ -133,13 +133,14 @@ def var_path(name):
 
 # An configuration argument.
 class CodalabArg(object):
-    def __init__(self, name, help, type=str, env_var=None, flag=None, default=None):
+    def __init__(self, name, help, type=str, env_var=None, flag=None, default=None, frontend=False):
```

Comment:
Let's add a docstring here. Mainly, we just need somewhere to document what `frontend` means.

Reply:
Added. Let me know what you think.

Reply:
Let's also talk about the fact that the frontend argument indicates that the env variable will be passed down to the frotnend container at build time, prefixed by REACT_APP_.

Reply:
Updated, thanks.